daml_dit_if/main/integration_context.py

```diff
@@ -109,6 +110,7 @@ class IntegrationContext:
 
     def __init__(self,
                  network: 'Network',
+                 run_as_party: 'Optional[str]',
```

Comment:
This really should be an `act_as`/`read_as` pair (and lists) to match the API for multi-party submissions. I anticipate being able to act as a member of a group as pretty important for integrations (and bots) going forward.

Reply:
(and of course sadly you won't be able to feed this in the current version of dazl…but it's coming!)

Reply:
ooo @da-tanabe this also implies artifact model better to separate `run_as` into `read_as/act_as` for all automation kinds (pybot, trigger, integrations)

Reply:
Agreed... but... No. Not now. Later.

The major point of this PR is that 'integrations now work like bots and triggers with respect to party arguments'.

Breaking that to support a feature that doesn't yet exist in either DAZL or DABL seems very, very wrong.

https://github.com/DACH-NY/daml-cloud/blob/master/artifact-service/daml/DAMLHub/ArtifactService/V004/Companions/Records.daml#L128

Reply:
@lima-da beat me to it.

Reply:
Well the reason why I bring it up now…are we changing the _external_ API that people are interacting with?

I'd rather change it once than twice (especially in the span of just a few weeks).